py-console/xkdb.py

```diff
@@ -231,9 +231,12 @@ def main():
     parser.add_argument('--type', '-t', '--class', '-c', dest='type', 
                         action='store', default='quark',
                         help='the type of backend board to connect to (default=quark)')
-    parser.add_argument('--xinu', '-x', dest='xinu_file', action='store', default='xinu',
+    parser.add_argument('--xinu', '-x', dest='xinu_file', action='store', default='xinu.xbin',
```

Comment:
The default should remain xinu.elf since that is the default in the xinu source: https://github.com/real-xinu/xinu-bbb/blob/master/compile/Makefile#L19.

Reply:
Changed this back in fixup commit!